File: Week 8 -no pset for Week 6 and 7/Mario/mario.py
import cs50

while True:
    print ("Height:", end="")
    height = cs50.get_int()
    if height>0 and height<25:
        break
    
for row in range (height):
    for left_space in range (height-row-1):
        print(" ", end="")
    for left_hash in range(row+1):
        print("#", end="")
    for right_space in range (2):
        print (" ", end="")
    for right_hash in range(row+1):
        print("#", end="")
    print(" ")

File: Week 1/mario.c
#include <stdio.h>
#include <cs50.h>

int main (void)
{
    int h;     
    do
    {
     printf("Height:");
     h= get_int();   
    } 
    while (h<0||h>=24);
       
    for (int i=0; i<h; i++)
    {
        for (int a=0; a<h-i-1; a++)
        {
            printf(" ");
        }
        for (int b=0; b<i+1; b++)
        {
            printf ("#");
        }
        
        for (int c=0; c<2; c++)
        {
            printf (" ");
        }
        
         for (int d=0; d<i+1; d++)
        {
            printf ("#");
        }
        
        printf("\n");
    }
}
